app/assets/javascripts/controllers/container_dashboard/charts/heatmap/heatmap-directive.js

```diff
@@ -0,0 +1,124 @@
+angular.module('miq.charts').directive('heatMap', ['$timeout',
+  function($timeout) {
```

Comment:
You don't have to wait for @skateman , as jeff said, angular-patternfly 2.6 doesn't depend on patternfly 2.6, and angular-patterfly is out already..